Make.config

```diff
@@ -40,9 +43,9 @@ IOS_PACKAGE_VERSION_BUILD=$(IOS_COMMIT_DISTANCE)
 IOS_PACKAGE_UPDATE_ID=$(shell printf "2%02d%02d%02d%03d" $(IOS_PACKAGE_VERSION_MAJOR) $(IOS_PACKAGE_VERSION_MINOR) $(IOS_PACKAGE_VERSION_REV) $(IOS_PACKAGE_VERSION_BUILD))
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=10.2
-XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_10.2.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode102.app/Contents/Developer
+XCODE_VERSION=11.0
+XCODE_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_11_GM_Seed.xip
+XCODE_DEVELOPER_ROOT=/Applications/Xcode11-GM.app/Contents/Developer
 
 XCODE94_VERSION=9.4
 XCODE94_URL=http://xamarin-storage/bot-provisioning/xcodes/Xcode_9.4.xip
```

Comment:
is there a point in keeping 9.4 around ?
can be fixed in a subsequent PR

Reply:
It's on my todo list to remove the Xcode 9.4 dependency, I just didn't want to complicate this PR even more.

Reply:
@spouliot AFAIK I added a check but just for Catalina.